core/src/main/java/io/questdb/cutlass/http/processors/TextQueryProcessor.java

```diff
@@ -361,6 +361,10 @@ private boolean parseUrl(
             skip = 0;
         }
 
+        if (configuration.getMaxQueryResponseRowLimit() != Long.MAX_VALUE && (stop - skip) > configuration.getMaxQueryResponseRowLimit()) {
```

Comment:
comparison to MAX_VALUE seems redundant

Reply:
I would also try to use min/max functions, in case Java can JIT those without branches